src/main/java/Type.java

```diff
@@ -0,0 +1,3 @@
+public enum Type {
+    TODO, DEADLINE, EVENT
```

Comment:
I think it was excellent design thinking on your part to come up with this enum, great job! My suggestion is to make the name of this enum more descriptive, eg. "TaskType" instead of "Type".